include/frg/bitset.hpp

```diff
@@ -0,0 +1,248 @@
+#ifndef FRG_BITSET_HPP
+#define FRG_BITSET_HPP
+#include <cstddef>
+#include <cstdint>
+
+namespace frg {
+	namespace detail {
+		constexpr auto div_roundup(auto v1, auto v2) { return (v1 + v2 - 1) / v2; }
```

Comment:
Given that the two `auto` parameters basically turn this into a templated function, the signedness of those params is left to chance - which might result in undefined behavior (signed int overflow is undefined, while unsigned ints wrap around).

Reply:
Likely best to use size_t (and maybe handle the overflows)